Models/Connect/ProductToRemoteCategoryRepository.php

```diff
@@ -135,7 +135,7 @@ public function getRemoteCategoryIds($articleId)
         $builder = $this->createQueryBuilder('ptrc');
         $builder->select('ptrc.connectCategoryId');
         $builder->where('ptrc.articleId = :articleId');
-        $builder->setParameter('articleId', $articleId);
+        $builder->setParameter('articleId', (int) $articleId);
```

Comment:
is this enough to make doctrine do an int compare or do you need?:
`$builder->setParameter('articleId', (int) $articleId, PDO::PARAM_INT);`